doc/rfd/rfd/0009/README.md

```diff
@@ -0,0 +1,176 @@
+---
+authors: Victor Cuadrado <vcuadradojuan@suse.com>, Ettore Di Giacinto <edigiacinto@suse.de>
+state: discussion
+discussion: https://github.com/cloudfoundry-incubator/kubecf/pull/1495
+---
+
+# RFD 9: Specification for deployment and automation scripts
+
+## Context
+
+
+Dealing with a handful of loosely dependent Helm charts and building a Helm
+distribution from them without creating an all-or-nothing "super-chart" is
+difficult. One needs to make sure that deployment and automation scripts are
+interoperable between charts, repos, repos' histories, and that they can be
+reused and extended downstream as needed.
+
+At the same time one needs to create, reuse, and extend test environments across
+charts, and between upstream and downstream.
+
+The following spec is an interface specification for sharing deployment and
+automation scripts of Helm charts and utility applications.
+
+
+## Specification
+
+### Version `0.1`.
+
+
+The interface consists in the following structure of folders and executables,
+together with their expected use:
+
+```
+├── clean/*
+├── chart/*
+├── configure/*
+│   └── patches/*
+├── install/*
+├── wait/*
+├── include/*
+│
+├── one-offs/*
+│
+├── include/*
+├── example-values/*
+│
+└── spec_ver
+```
+
+This structure as a whole is encouraged to be published together with the Helm
+chart. This can be for example as artifacts of a chart release, or present in
+the chart repository, but we recommend that this files are bundled inside of the
+chart: this simplifies consumption downstream.
+
+While targets are allowed to take arguments and options on execution besides the
```

Comment:
Can you please define `target` before this paragraph?

Reply:
done! thanks